SonarQube.Bootstrapper/BootstrapperSettings.cs

```diff
@@ -20,6 +20,12 @@ public class BootstrapperSettings : IBootstrapperSettings
         public const string BuildDirectory_TFS2015 = "AGENT_BUILDDIRECTORY";
 
         /// <summary>
+        /// The logical version of the boostrapper API that has to match the logical version of the preprocessor / postprocessor. 
```

Comment:
Typo - "boostrapper".
"The logical version of the bootstrapper API that the pre/post-processor must support" is more accurate.